Clipy/Sources/AppDelegate.swift

```diff
@@ -96,8 +96,16 @@ class AppDelegate: NSObject {
             return
         }
 
-        AppEnvironment.current.pasteService.copyToPasteboard(with: clip)
-        AppEnvironment.current.pasteService.paste()
+        let modifierFlags = NSEvent.modifierFlags()
+
+        switch modifierFlags {
```

Comment:
`NSEventModifierFlags` is not enum classes. Please replace
```
if modifierFlags.contains(.option) {
    AppEnvironment.current.clipService.clear(clip: clip)
} else {
    AppEnvironment.current.pasteService.copyToPasteboard(with: clip)
    AppEnvironment.current.pasteService.paste()
}
```

Reply:
That's strange. I built this against Xcode 9 beta and it works. I've just checked with Xcode 8.3 and it fails. Presumable this is due to a change between Swift 3 and 4.

The replacement above would result in any combination of modifier keys that included `.option` (i.e. a superset) would trigger the delete action. From my quick testing with other apps that appears to not be how shortcuts work. Shortcuts seems to only fire if the exact set of modifier keys are pressed. Therefore I've changed the code to have changed the switch case to `[.option]`.